src/apps/chat/models.py

```diff
@@ -13,6 +13,22 @@
 from model_utils.models import TimeStampedModel
 
 
+class Category(models.Model):
```

Comment:
Categories should be user-specific. So the User can organize his chats into folders.